packages/client/src/components/UploadForm.tsx

```diff
@@ -0,0 +1,77 @@
+import { authFetch } from '@/helpers';
+import { AuthState } from '@/types';
+import { useState } from 'react';
+import { SignedUrlResponseBody } from '@guardian/transcription-service-common';
+
+const uploadToS3 = async (url: string, blob: Blob) => {
+	try {
+		const response = await fetch(url, {
+			method: 'PUT',
+			body: blob,
+		});
+		const status = await response.status;
+		console.log('upload success:', status);
```

Comment:
We don't have any client side logging capture stuff so probably not worth logging the happy path.